NuRadioReco/examples/RNOG/rnog_config.yaml

```diff
@@ -11,7 +11,7 @@ readRNOGDataMattak:
     # Use mattak to calibrate data on the fly. Requires calibration files to be available
     read_calibrated_data: False
     # Do not apply a baseline correction in the read module, do it afterwards
-    apply_baseline_correction: null
+    apply_baseline_correction: none
```

Comment:
so `null` converts to `None` and I thought I had added `None` as possilbe value for this module. Maybe it did not yet make it into your branch, sorry for this.